libtelemetry/src/full/java/com/mapbox/android/telemetry/MapboxTelemetry.java

```diff
@@ -67,13 +69,27 @@ public MapboxTelemetry(Context context, String accessToken, String userAgent) {
     this.queue = EventsQueue.create(this, executorService);
   }
 
+  public MapboxTelemetry(Context context, String accessToken) {
+    initializeContext(context);
+    setAccessToken(context, accessToken);
+    AlarmReceiver alarmReceiver = obtainAlarmReceiver();
+    this.schedulerFlusher = new SchedulerFlusherFactory(applicationContext, alarmReceiver).supply();
+    this.telemetryEnabler = new TelemetryEnabler(true);
+    initializeTelemetryListeners();
+    initializeAttachmentListeners();
+    // Initializing callback after listeners object is instantiated
+    this.httpCallback = getHttpCallback(telemetryListeners);
+    this.executorService = ExecutorServiceFactory.create("MapboxTelemetryExecutor", 3,
+      20);
+    this.queue = EventsQueue.create(this, executorService);
+  }
+
   // For testing only
-  MapboxTelemetry(Context context, String accessToken, String userAgent, EventsQueue queue,
+  MapboxTelemetry(Context context, String accessToken, EventsQueue queue,
                   TelemetryClient telemetryClient, Callback httpCallback, SchedulerFlusher schedulerFlusher,
                   Clock clock, TelemetryEnabler telemetryEnabler, ExecutorService executorService) {
     initializeContext(context);
     setAccessToken(context, accessToken);
-    this.userAgent = userAgent;
```

Comment:
We can probably remove this from the deprecated constructor as well as the unused variable